website/client/src/components/tasks/spells.vue

```diff
@@ -267,18 +267,16 @@ export default {
       );
     },
     spellDisabled (skill) {
-      if (skill === 'frost' && this.user.stats.buffs.streaks) {
-        return true;
+      let isSpellDisabled = false;
+
+      if ((skill === 'frost' && this.user.stats.buffs.streaks)
```

Comment:
Especially if we add more conditions here in future I think this would look clearer if we do something like 

```
    spellDisabled (skill) {
        if (skill === 'frost' && this.user.stats.buffs.streaks) return true;
        if (skill === 'stealth' && this.user.stats.buffs.stealth >= this.user.tasksOrder.dailys.length) return true;

       return false

Reply:
Thanks @paglias ! Good point on the extensibility of those conditions, I'll update that. I'll likely have time to work on tests next weekend.

It sounds like #6867 will only be partially solved by this PR then.

Reply:
> The tooltip should say `Number of dailies avoided: X` like it does now or `Number of dailies that will be avoided: X`? The current version seems to imply that they've already been avoided

Ah yes, `Number of dailies that will be avoided: X` is better!